cmd/utils.go

```diff
@@ -40,7 +40,7 @@ type DependencyOverview struct {
 	MainModuleName string
```

Comment:
I think this should change to a []string and be set directly from the mainModules input.

If len(mainModules) == 0, then it can be set to the first LHS in the `mod graph` output, as it is now